sirepo/template/openmc.py

```diff
@@ -673,6 +704,22 @@ def _generate_run_mode(data, v):
     return f"openmc.run(threads={cores})"
 
 
+def _generate_save_weight_windows(data):
+    if data.models.settings.varianceReduction in (
+        "weight_windows_mesh",
+        "weight_windows_tally",
+    ):
+        return f"""
+import sirepo.sim_data
+sirepo.sim_data.get_class("{ SIM_TYPE }").put_sim_file(
```

Comment:
I think it might make sense to create a put_lib_file interface. The file is only read from the sim dir just to copy it to the lib dir.

Reply:
In this case, it is an option to copy the sim file to the lib directory. The user may not want to use the computed file for other simulations.